src/render.ts

```diff
@@ -285,6 +293,7 @@ let previousScreen = ''
 let hover = 0
 let fade = 0
 titleEsa.x = 20
+let selectedCharacter = [0, 1]
```

Comment:
I'd call this `selectedCharacters`, since it's an array that usually contains more than one element. But I leave that for you to decide 😄